packages/components/src/styles/display/display.stories.ts

```diff
@@ -40,17 +40,17 @@ export const Default = {
 /**
  * Use the `&--size-*` classes for alternative appearances.
 
- * - `sd-display--size-4xl`: 4xl can be used as an alternative
+ * - `sd-display`: 4xl is the default display size
  * - `sd-display--size-3xl`: 3xl can be used as an alternative
- * - xl is the default display size
+ * - `sd-display--size-xl`: xl can be used as an alternative
```

Comment:
```suggestion
 * - `sd-display--size-xl`
```

Reply:
Did we removed the explanation of different variants/classes etc.?